cluster/charts/crossplane/values.yaml.tmpl

```
@@ -95,6 +95,10 @@ podSecurityContextCrossplane: {}
 
 podSecurityContextRBACManager: {}
 
+extraVolumesCrossplane: {}
+
+extraVolumeMountsCrossplane: {}
```

```suggestion
extraVolumesCrossplane: []

extraVolumeMountsCrossplane: []
```

Aren't these arrays? Similar to what @bobh66 fixed recently in https://github.com/crossplane/crossplane/pull/3832

---

@turkenh this is slightly different than that case because we are not using these values as a sequence. We iterate over the `args` but with these volume values we treat them as a YAML block.